StPetersburg/RNBefns11.xml

```diff
@@ -43,15 +43,17 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      <locus from="11" to="195"/>
                      <title type="complete" ref="LIT2293Sewas" xml:lang="gez">
                              </title>
-                     <textLang mainLang="gez"/>
+                     <textLang mainLang="gez" otherLangs="am"/>
+                     <note>Gǝʿǝz-Amaric entiers are supplied with Arabic translations
```

Comment:
Amharic

Reply:
Arabic not encoded?

Reply:
How can I  encode a third language? I can encode only two under textLang; I do not see the text and do not understand how it is arranged.

Reply:
Ok, I see.